### PR TITLE
New 42 serializer for vacancies model kovalchuk

### PR DESCRIPTION
**Pull request by Maksson**

### CODE REVIEW

**Review by MetalMari — APPROVED** [review]

File: codeArena/vacancies/serializers.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from django.db.models import fields
+from rest_framework import serializers
+from rest_framework.validators import UniqueValidator
```
Comment (MrSancho42) [inline codeArena/vacancies/serializers.py]: You not using this import, so you can remove it

File: codeArena/vacancies/urls.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+from .views import CreateVacanciesView, VacanciesView
+
+urlpatterns = [
+    path('', VacanciesView.as_view()),
+    path('create_vacancies/', CreateVacanciesView.as_view()),
+]
```
Comment (MrSancho42) [inline codeArena/vacancies/urls.py]: Missed empty string

**Review by OleksandrHavron — APPROVED** [review]

File: codeArena/vacancies/views.py [hunk 3]
```diff
@@ -1,3 +1,21 @@
...
+from rest_framework import status
+from django.db.utils import DatabaseError
 
-# Create your views here.
+class VacanciesView(APIView):
+
+    def get(self, request):
+        vacancies = Vacancies.objects.all()
+        serializer = VacanciesSerializer(vacancies, many=True)
+        return Response(serializer.data)
```
Comment (MrSancho42) [inline codeArena/vacancies/views.py:12]: You not returned HTTP status

File: codeArena/vacancies/views.py [hunk 4]
```diff
@@ -1,3 +1,21 @@
...
+
+class CreateVacanciesView(APIView):
+
+    def post(self, request, format='json'):
+        vacancy = CreateVacanciesSerializer(data=request.data)
+        if vacancy.is_valid(raise_exception=True):
+            vacancy1 = vacancy.save()
+        return Response({"success": f'News {vacancy1.name} created successfully'})
```
Comment (MrSancho42) [inline codeArena/vacancies/views.py]: Same problem as highter

File: codeArena/vacancies/views.py [hunk 5]
```diff
@@ -1,3 +1,21 @@
-from django.shortcuts import render
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from .serializers import CreateVacanciesSerializer, VacanciesSerializer
+from .models import Vacancies
+from rest_framework import status
+from django.db.utils import DatabaseError
```
Comment (MrSancho42) [inline codeArena/vacancies/views.py]: Where you use DatabaseError?

File: codeArena/vacancies/views.py (hunk 3 above)
Comment (vvelizariy) [reply codeArena/vacancies/views.py:12]: Actually, if response code 200 is ok, then can be skipped. Default return code is 200.

**Review by vvelizariy — APPROVED** [review]

**Pull request merged**